tasks.py

```diff
@@ -53,13 +54,23 @@ def start():
 
 
 @task
-def start_for_tests():
-    if os.path.exists(DB_FILE_PATH):
-        os.remove(DB_FILE_PATH)
-
-    OSF_DIR = '~/Desktop/OSF'
-    if os.path.exists(OSF_DIR):
-        shutil.rmtree(OSF_DIR)
+def start_for_tests(dropdb=True, droplog=False, dropdir=False):
```

Comment:
add option to change the polling_delay timer and add logging options since the dev_mode is not quite working with local copy of osf. will make the change when I merge.